spring-native-configuration/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientHints.java

```diff
@@ -30,6 +30,7 @@
 import org.springframework.nativex.hint.TypeHint;
 import org.springframework.nativex.type.NativeConfiguration;
 
+// TODO Not sure AccessBits.ALL is needed on these, to be checked
 @NativeHint(trigger = LoadBalancerEurekaAutoConfiguration.class, types = {
```

Comment:
Did you try without those hints on `EurekaLoadBalancerClientConfiguration` and `EurekaReactiveDiscoveryClientConfiguration`? In theory that should not be needed.